packages/cli/src/ceramic-daemon.ts

```diff
@@ -143,6 +152,7 @@ class CeramicDaemon {
   registerAPIPaths (app: core.Express, gateway: boolean): void {
     app.get(toApiPath('/records/:docid'), this.records.bind(this))
     app.post(toApiPath('/documents'), this.createDocFromGenesis.bind(this))
+    app.post(toApiPath('/states'), this.multiQuery.bind(this))
```

Comment:
Maybe we should call this endpoint `multiqueries` since we no longer have a `state` endpoint?
cc @simonovic86 any thoughts on naming here?

Reply:
yep, `multiqueries` is more appropriate 👍

Reply:
Probably `multiquery` is better?

Reply:
I think `multiqueries` makes more sense:
- we are creating one multiquery in for multiqueries resource
- we used plural before for all the resources
- we can refactor it to make it async and query the multiquery itself (GET `/multiqueries/:id`)